vantage6/tools/docker_wrapper.py

```diff
@@ -44,7 +44,17 @@ def docker_wrapper(module: str):
     output_file = os.environ["OUTPUT_FILE"]
     info(f"Writing output to {output_file}")
     with open(output_file, 'wb') as fp:
-        fp.write(pickle.dumps(output))
+        try:
+            output_format = input_data['output_format']
+            # Indicate output format
+            fp.write(output_format.encode() + b'.')
+
+            # Write actual data
+            serialized = serialization.serialize(output, output_format)
+            fp.write(serialized)
+        except KeyError:
```

Comment:
I changed the check. Instead of try-except I check whether the key is present in `input_data`.

Whether the output_format is actually valid will be handled by the `DataFormat` enum.

Reply:
👍